src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DJApplicantReceivedNotificationHandler.java

```diff
@@ -68,10 +71,16 @@ private String identifyTemplate(CaseData caseData) {
             template = notificationsProperties.getApplicantSolicitor1DefaultJudgmentRequested();
             templateReference = REFERENCE_TEMPLATE_REQUESTED;
         }
-        if (ofNullable(caseData.getRespondent2()).isEmpty()) {
+        if (ofNullable(caseData.getRespondent2()).isEmpty() && !caseData.isLipvLipOneVOne()) {
             template = notificationsProperties.getApplicantSolicitor1DefaultJudgmentReceived();
             templateReference = REFERENCE_TEMPLATE_RECEIVED;
         }
+        if (ofNullable(caseData.getRespondent2()).isEmpty()
+            && caseData.isLipvLipOneVOne()
```

Comment:
isn't caseData.isLipvLipOneVOne() already checking that there is no respondent2? because it onevone

Reply:
Done!